lib/gemstash/storage/local_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "fileutils"
+
+module Gemstash
+  class Storage
+    class LocalService < BaseService # :nodoc:
```

Comment:
I think we don't need exits?, read and delete I feel we just coupling the a `Resource` with a service by forcing to using these service methods. What you think?